blog/content/post/json-2.md

```diff
@@ -0,0 +1,202 @@
+---
+date: "2016-07-23T14:46:01+07:00"
+draft: false
+title: "JSON decoding numbers"
+description: "Decode arbitrary JSON data contain number"
+tags: [ "golang", "json", "encode", "decode", "interface", "reflection"]
+categories:
+  - "Standard Library"
+  - "Reflection"
+---
+
+### Preface
+
+At previous post, we had encountered a problem when decode arbitray JSON data containing number. To recap, we decode JSON data into a `map[string]interface{}` and suffered type change, from `int` to `float64`, see more at [JSON Pitfalls]({{< relref "post/json-1.md" >}}).
```

Comment:
In previous post, we encountered ... when decoding arbitrary JSON data which contains number.
... we decoded